### ISSUE TITLE
download.topo() fails owing to server change

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1697]
```R
library(oce)
topo <- read.topo(download.topo(-80, -70, 23, 30))
```
yields
```
trying URL 'https://gis.ngdc.noaa.gov/cgi-bin/public/wcs/etopo1.xyz?filename=etopo1.grd&request=getcoverage&version=1.0.0&service=wcs&coverage=etopo1&CRS=EPSG:4326&format=gmt&resx=0.066667&resy=0.066667&bbox=-80.000000,23.000000,-70.000000,30.000000'
Error in download.file(url, destination) : 
  cannot open URL 'https://gis.ngdc.noaa.gov/cgi-bin/public/wcs/etopo1.xyz?filename=etopo1.grd&request=getcoverage&version=1.0.0&service=wcs&coverage=etopo1&CRS=EPSG:4326&format=gmt&resx=0.066667&resy=0.066667&bbox=-80.000000,23.000000,-70.000000,30.000000'
```

**Comment by dankelley** [comment]
Hm, `dc::dc.topo()` also fails, and on a call (shown below) that worked 3 days ago.  Either the server changed something, or this is a temporary thing.
```R
library(dc)
topoFile <- dc.topo(-81, -72, 22, 31, resolution=1, force=TRUE)
```

**Comment by dankelley** [comment]
For comparison, here is what `marmap::getNOAA.bathy()` uses in composing the query string:
```R
        WEB.REQUEST <- paste0("https://gis.ngdc.noaa.gov/arcgis/rest/services/DEM_mosaics/ETOPO1_bedrock/ImageServer/exportImage?bbox=", 
            x1, ",", y1, ",", x2, ",", y2, "&bboxSR=4326&size=", 
            ncell.lon, ",", ncell.lat, "&imageSR=4326&format=tiff&pixelType=S16&interpolation=+RSP_NearestNeighbor&compression=LZW&f=image")
        dat <- suppressWarnings(try(raster::
```

**Comment by dankelley** [comment]
Oh, it seems that NOAA changed the API recently ... check out the date of the top item in the `marmap` status log:
```
commit 97fb03998b5e9208b241df4cebd9543231bb9e57 (HEAD -> master, origin/master, origin/marmap-v1.0.4, origin/HEAD)
Author: Benoit Simon-Bouhet <besibo@gmail.com>
Date:   Tue May 19 14:41:07 2020 +0200

    Fixed getNOAA.bathy() to (i) take into account the change in NOAA's API for downloading bathymetric data and (ii) allow for downloading bathy data to or reading bathy data from a user defined file path
```

**Comment by dankelley** [comment]
Although it's annoying that NOAA changed the URL structure, at least they now -- finally -- have an API!!!! https://gis.ngdc.noaa.gov/arcgis/sdk/rest/

**Comment by dankelley** [comment]
I see how to fix this (as in issue dankelley/dc#15 ) and will port that method here within a week or so.

**Comment by dankelley** [comment]
Fixed in "develop", e.g.  the `?download.topo` example, shown below (the white bands result from the fact that the specified lon-lat box is not square when plotted at the aspect ratio of the centre of the image).

```R
library(oce)
topoFile <- download.topo(west=-66, east=-60, south=43, north=47,
                          resolution=1, destdir="~/data/topo")
topo <- read.topo(topoFile)
imagep(topo, zlim=c(-400, 400), drawTriangles=TRUE)
if (requireNamespace("ocedata", quietly=TRUE)) {
    data(coastlineWorldFine, package="ocedata")
    lines(coastlineWorldFine[["longitude"]], coastlineWorldFine[["latitude"]])
}
```
![dandan](https://user-images.githubusercontent.com/99469/83354623-4f27fd80-a330-11ea-86be-ae534cc7b766.png)
